pkg/storage/storagebackend/config.go

```diff
@@ -26,7 +26,7 @@ const (
 
 // Config is configuration for creating a storage backend.
 type Config struct {
-	// Type defines the type of storage backend, e.g. "etcd2", etcd3". Default ("") is "etcd2".
+	// Type defines the type of storage backend, e.g. "etcd2", etcd3". Default ("") is "etcd3".
```

Comment:
As long as I'm probably fine with changing the default for tests in this PR, I don't think we should do it for binaries (since we still don't have rollback etc., so we may not able to launch it).